bin/fuel-core/src/cli/snapshot.rs

```diff
@@ -240,26 +277,29 @@ fn load_chain_config(
     Ok(chain_config)
 }
 
-fn open_db(path: &Path) -> anyhow::Result<Database> {
-    Database::<OnChain>::open_rocksdb(path, None)
+fn open_db(path: &Path) -> anyhow::Result<CombinedDatabase> {
+    CombinedDatabase::open(path, 1024 * 1024 * 1024)
```

Comment:
Do we need to make `1GB` configurable?

Reply:
Don't know but I made it a parameter with a default.